kml/types.hpp

```diff
@@ -503,4 +502,10 @@ inline std::string DebugPrint(BookmarkIcon icon)
 {
   return ToString(icon);
 }
+
+inline std::string DebugPrint(TimestampMillis const & ts)
+{
+  return DebugPrint(static_cast<Timestamp>(ts));
```

Comment:
Compiler shows warning: `all paths through this function will call itself` because types `Timestamp` and `TimestampMillis` could be converted to one another. Is there any way to force use `DebugPrint(Timestamp)` function from `base/internal/message.hpp` ?